src/components/views/room_settings/RoomPublishSetting.tsx

```diff
@@ -49,8 +58,10 @@ export default class RoomPublishSetting extends React.PureComponent<IProps, ISta
 
         client
             .setRoomDirectoryVisibility(this.props.roomId, newValue ? Visibility.Public : Visibility.Private)
-            .catch(() => {
+            .catch((err) => {
```

Comment:
Did you mean to log this error maybe? Nothing happens with this param.

Reply:
yeah, was added to console log, removing it